README.md

```diff
@@ -8,16 +8,17 @@ This repository contains samples for usage of the Google Cloud Machine Learning
 
 ### CPU and GPU
 
-* [Keras Census](census/keras) - Keras single node model
-* [Census](census) - TensorFlow census models
-  * [Canned Estimator](census/estimator) - TensorFlow canned estimator model
-  * [Custom Estimator](census/customestimator) - TensorFlow custom estimator model
+* [Census](census) - Predict a person's income level
+  * [Keras Census](census/keras) - [Keras](https://keras.io/) single node model
+  * [Canned Estimator](census/estimator) - [TensorFlow canned estimator](https://www.tensorflow.org/programmers_guide/estimators#pre-made_estimators) model
+  * [Custom Estimator](census/customestimator) - [TensorFlow custom estimator](https://www.tensorflow.org/programmers_guide/estimators#custom_estimators) model
   * [Low Level TF](census/tensorflowcore) - TensorFlow low level API model
-* [Criteo](criteo_tft) - Advertisement CTR prediction on Criteo dataset
-* [Flowers](flowers) - Image transfer learning using flowers dataset
-* [Movielens](movielens) - Personalized recommendation using movielens dataset
-* [Reddit](reddit_tft) - Wide and Deep model using reddit dataset
-* [Cifar10](https://github.com/tensorflow/models/tree/master/tutorials/image/cifar10_estimator) - Multi-GPU Cifar10
```

Comment:
Also the links are pointing to your repo.

Reply:
I think they are just pointing there because the PR has you view the branch on my repo. They are relative links, so will point at the official repo once the PR is merged.